srcs/vm/main.c

```diff
@@ -23,8 +23,9 @@ int			main(int argc, char **argv)
 	t_arena		arena;
 
 	ft_bzero(&arena, sizeof(arena));
+	arena.dump_flag = -1;
```

Comment:
Waarom staat dit er in, arena is al 0, waarom -1?

Reply:
With [-dump nbr_cycles] nbr_cycles can be 0, I can be wrong?

Reply:
nbr_cycles mag 0 zijn, als dump_flag niet op -1 wordt geset kan dit niet worden gecontroleerd

Reply:
Heb dit zelf al in check_args aangepast dus dubbelop